devel/ci/integration/rabbitmq/rabbitmqadmin

```diff
@@ -0,0 +1,1050 @@
+#!/usr/bin/env python3
+
+#   The contents of this file are subject to the Mozilla Public License
+#   Version 1.1 (the "License"); you may not use this file except in
+#   compliance with the License. You may obtain a copy of the License at
+#   http://www.mozilla.org/MPL/
+#
+#   Software distributed under the License is distributed on an "AS IS"
+#   basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+#   License for the specific language governing rights and limitations
+#   under the License.
+#
+#   The Original Code is RabbitMQ Management Plugin.
+#
+#   The Initial Developer of the Original Code is GoPivotal, Inc.
+#   Copyright (c) 2007-2016 Pivotal Software, Inc.  All rights reserved.
+from __future__ import print_function
+
+try:
+    from signal import signal, SIGPIPE, SIG_DFL
+    signal(SIGPIPE, SIG_DFL)
+except ImportError:
+    pass
+
+import sys
+if sys.version_info[0] < 2 or (sys.version_info[0] == 2 and sys.version_info[1] < 6):
+    eprint("Sorry, rabbitmqadmin requires at least Python 2.6 (2.7.9 when HTTPS is enabled).")
+    sys.exit(1)
+
+from optparse import OptionParser, TitledHelpFormatter
+import urllib
+import base64
+import json
+import os
+import socket
+import ssl
+import traceback
+
+if sys.version_info[0] == 2:
+    from ConfigParser import ConfigParser, NoSectionError
+    import httplib
+    import urlparse
+    from urllib import quote_plus
+    from urllib import quote
+    def b64(s):
+        return base64.b64encode(s)
+else:
+    from configparser import ConfigParser, NoSectionError
+    import http.client as httplib
+    import urllib.parse as urlparse
+    from urllib.parse import quote_plus
+    from urllib.parse import quote
+    def b64(s):
+        return base64.b64encode(s.encode('utf-8')).decode('utf-8')
+
+VERSION = '3.6.16'
```

Comment:
Can we link to where this file comes from?

Reply:
Or, would it make more sense to package it as an RPM, or maybe to just have the CI script download it from an authoritative source?

Reply:
Yeah that can be done, it's actually served from the RabbitMQ server itself.